Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NICOLE.MSG

```diff
@@ -188,7 +188,7 @@
 {284}{}{Can I ask a few more questions?}
 {285}{}{That was all for now. I gotta go.}
 
-{286}{Nic_73}{Please, we're busy. Don't you have things to do?}
+{286}{Nic_72}{Please, we're busy. Don't you have things to do?}

```

Comment:
Don't check on Nukapedia. The Fallout wikis are notoriously wrong about a lot of stuff. The only good source is raw vanilla files.

Reply:
Don't worry, I know that NIC_72.ACM is the right audio file. ;-)